lib/miq/ref_versions.rb

```diff
@@ -42,11 +42,11 @@ def build_index
     end
 
     def menus
-      @menus ||= begin
-        menus = Miq.site_dir.join("_data", "menus").glob("ref_menu_*.yml").sort.map(&:to_s)
-        latest = menus.index { |m| m.include?("ref_menu_latest.yml") }
-        menus << menus.delete_at(latest)
-      end
+      @menus ||=
+        Miq.menus_dir.glob("ref_menu_*.yml").sort.map(&:to_s).tap do |menus|
+          latest = menus.index { |m| m.include?("ref_menu_latest.yml") }
+          menus << menus.delete_at(latest) if latest
+        end
```

Comment:
This part is needed to deal with someone doing `bin/miq serve` directly but without building the docs.  Previously the menu files would exist and this would be fine, but now they won't exist unless one builds the docs first.